pkgs/development/python-modules/myjwt/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, click
+, colorama
+, cryptography
+, exrex
+, pyopenssl
+, pyperclip
+, questionary
+, requests
+, pytestCheckHook
+, pytest-mock
+, requests-mock
+}:
+
+buildPythonPackage rec {
+  pname = "myjwt";
+  version = "1.4.0";
+
+  src = fetchFromGitHub {
+    owner = "mBouamama";
+    repo = "MyJWT";
+    rev = version;
+    sha256 = "1n3lvdrzp6wbbcygjwa7xar2jnhjnrz7a9khmn2phhkkngxm5rc4";
+  };
+
+  # This is a work-around for possible release mismatches and to 
+  # not block other packages from building
+  postPatch = ''
+    substituteInPlace requirements.txt \
```

Comment:
This will probably be out of date by the next update. Maybe an patch so that we notice it easily?

Reply:
Changed to use a patch to remove the pinning